app/scripts/controllers/eventOverview.js

```diff
@@ -2,77 +2,79 @@
 
 angular
   .module('confRegistrationWebApp')
-  .controller('eventOverviewCtrl', function(
-    $rootScope,
-    $scope,
-    $location,
-    $route,
-    $http,
-    $sce,
-    ConfCache,
-    conference,
-  ) {
-    $rootScope.globalPage = {
-      type: 'admin',
-      mainClass: 'container event-overview',
-      bodyClass: '',
-      confId: conference.id,
-      footer: true,
-    };
+  .controller(
+    'eventOverviewCtrl',
+    (
+      $rootScope,
+      $scope,
+      $location,
+      $route,
+      $http,
+      $sce,
+      ConfCache,
+      conference,
+    ) => {
+      $rootScope.globalPage = {
+        type: 'admin',
+        mainClass: 'container event-overview',
+        bodyClass: '',
+        confId: conference.id,
+        footer: true,
+      };
 
-    $scope.resetImage = () => {
-      $scope.imageSrc = conference.image;
-      $scope.includeImageToAllPages = conference.includeImageToAllPages;
-    };
+      $scope.resetImage = () => {
+        $scope.imageSrc = conference.image;
+        $scope.includeImageToAllPages = conference.includeImageToAllPages;
+      };
 
-    $scope.conference = conference;
-    $scope.imageSrc = conference.image;
```

Comment:
Did you mean to remove this? I guess if you remove this the resetImage function will set it.